packages/receipt-renderer/src/render-escpos.ts

```
@@ -191,14 +195,29 @@ function walkNodes(
+			}
 			writeText(encoder, node.value, context.supportsCp932, context.normalizeText);
+			if (node.value) context.lineHasText = true;
 			break;
 		case 'text':
+			if (writeAlignedStandaloneTextLine(encoder, node.children, context)) {
+				break;
+			}
 			if (writeIndentedStandaloneTextLine(encoder, node.children, context)) {
 				break;
 			}
+			const previousAllowAlignedRawTextLine = context.allowAlignedRawTextLine;
```

Biome (2.4.14)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)